# HG changeset patch
# User Rafael Laboissiere <rafael@laboissiere.net>
# Date 1451996595 -3600
#      Tue Jan 05 13:23:15 2016 +0100
# Branch stable
# Node ID c27b6503e5d2bfbbdc7d58387857eb143c7b8c22
# Parent  f7e416862e909e83db265ccb5ddc518cf6908a12
etc/icons/octave.desktop.in: Add Keywords entry and field code to Exec.

Since mime types are registered, the Exec key must contain a field code
specifying a way to pass the filename as the parameter, see Desktop Entry
Specification for details. (bugs #46760, #46790)

http://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html

diff --git a/etc/icons/octave.desktop.in b/etc/icons/octave.desktop.in
--- a/etc/icons/octave.desktop.in
+++ b/etc/icons/octave.desktop.in
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Name=GNU Octave
 Comment=Scientific Computing using GNU Octave
 TryExec=%OCTAVE_PREFIX%/bin/octave
-Exec=%OCTAVE_PREFIX%/bin/octave --force-gui
+Exec=%OCTAVE_PREFIX%/bin/octave --force-gui %f
 Icon=%OCTAVE_IMAGEDIR%/octave-logo.svg
 Terminal=false
 Type=Application
 Categories=Education;Science;Math;
 StartupNotify=false
 MimeType=text/x-octave;text/x-matlab;
+Keywords=science;math;matrix;numerical computation;plotting;
